test/e2e/BUILD

```diff
@@ -240,6 +237,14 @@ go_library(
     ],
 )
 
+genrule(
```

Comment:
I can leave this genrule out if we don't want to preserve the name of the test binary.